### ISSUE TITLE
error SyntaxError

### ISSUE THREAD

**Issue by abdullahainun** [d3/d3-array#216]
I used this dependency for react native projects, before yesterday I'm building apk with no issue after I getting this. anyone can help me?

```bash
error SyntaxError: react-native-repo/node_modules/d3-array/dist/d3-array.js: Unexpected token, expected "," (104:6)

  102 |   var sum = 0, index = 0;
  103 |   return Float64Array.from(values, valueof === undefined
> 104 |     ? v => (sum += +v || 0)
      |       ^
  105 |     : v => (sum += +valueof(v, index++, values) || 0));
  106 | }

```

**Comment by mbostock** [comment]
Related https://github.com/FormidableLabs/victory-native/issues/653

**Comment by abdullahainun** [comment]
thank u very much, issue solved when I using "d3-array": "2.3.3", version

Pada tanggal Sel, 15 Jun 2021 pukul 21.27 Mike Bostock <
***@***.***> menulis:

**Comment by mbostock** [comment]
Also duplicate of #147.
